### PR TITLE
fix: Infinite scroll

### PR DESCRIPTION
**Pull request by drazik**
We messed with the DOM structure, breaking the infinite scroll. I didn't find precisely when. I remember we did some refactoring on the balance history chart someday, but can't find the PR. And I didn't want to lose too many time looking for it. If you think it's important to find it, I can do more research.

### CODE REVIEW

**Comment by kosssi** [comment]
I don't understand the fix, can you explain it ?

**Comment by kosssi** [comment]
Changing the structure of the DOM makes it possible to re-run the scroll at the right place 👍

File: src/ducks/transactions/TransactionHeader.jsx [hunk 1]
```diff
@@ -8,9 +8,19 @@ import { Breadcrumb } from 'components/Breadcrumb'
 import { ConnectedSelectDates } from 'components/SelectDates'
 import { AccountSwitch } from 'ducks/account'
 import TransactionSelectDates from 'ducks/transactions/TransactionSelectDates'
+import { TransactionTableHead } from './Transactions'
+import flag from 'cozy-flags'
+import HistoryChart from 'ducks/balance/HistoryChart'
```
Comment (ptbrowne) [inline src/ducks/transactions/TransactionHeader.jsx]: Au final on pourrait avoir le TransactionTableHead ici non ? Au lieu de l'importer ?

Comment (drazik) [reply src/ducks/transactions/TransactionHeader.jsx]: Yep, je fais cette modification

**Pull request merged**